src/styles/apps-menu.styl

```diff
@@ -1,10 +1,14 @@
+@require 'settings/breakpoints'
+
 .apps-menu-grid
     display flex
     flex-wrap wrap
     width 100%
     box-sizing border-box
-    padding-left .5rem
-    padding-right .5rem
+
+    +desktop()
```

Comment:
be careful with this because I think we have to consider the tablet as the desktop 🤔

Reply:
It is validated by Swann